tests/snapshots/build-p.js.md

```diff
@@ -63,6 +63,9 @@ Generated by [AVA](https://ava.li).
             'string',
           ],
         },
+        _overrides: {
```

Comment:
I would prefer it if we moved the snapshot updates to a separate commit for this whole branch.